### PR TITLE
Add docker stack for e2e tests, remove cypress

### PR DESCRIPTION
**Pull request by MattyMay**
~~Add e2e testing configuration and e2e tests for the project settings page~~
Add docker stack for e2e tests & remove cypress

### CODE REVIEW

File: cypress/e2e/admin/project_settings.cy.ts [hunk 1]
```diff
@@ -0,0 +1,804 @@
...
+      .should("be.visible")
+      .click();
+
+    cy.save()
+      .get_api_errors()
+      .should("have.length", 1)
+      .first()
+      .should("contain.text", "closing time");
+
+    // make sure error goes away after valid request
```
Comment (james-perretta) [inline cypress/e2e/admin/project_settings.cy.ts]: This suggests that there would be a check such as `.get_api_errors().should("have.length", 0)`, but I don't see that below. Should there be?

File: cypress/e2e/admin/project_settings.cy.ts [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+      .should("be.disabled")
+      .click({ force: true })
+      .get_by_testid(ElementId.max)
+      .should("be.visible")
+      .type("b0")
+      .get_by_testid(ElementId.max_err)
+      .should("contain.text", "integer")
+      .get_by_testid(ElementId.save)
+      .should("be.disabled")
+      .click({ force: true })
```
Comment (james-perretta) [inline cypress/e2e/admin/project_settings.cy.ts]: Is this basically saying "act as if the user clicked the mouse on the disabled button", with the expectation that nothing happens (i.e., the form is not submitted)?

File: cypress/e2e/admin/project_settings.cy.ts (hunk 1 above)
Comment (MattyMay) [reply cypress/e2e/admin/project_settings.cy.ts]: Ah this isn't very clear. `save_and_reload()` verifies that there are no API errors after saving and after reloading. It's mentioned in the jsdoc but maybe I should rename the action? Or get rid of the action and just call `save().get_api_errors() ... refresh().get_api_errors() ...` ?

File: cypress/e2e/admin/project_settings.cy.ts (hunk 2 above)
Comment (MattyMay) [reply cypress/e2e/admin/project_settings.cy.ts]: Yes. Maybe a silly thing to test 🤷

**Review by james-perretta — APPROVED** [review]

**Pull request merged**